types/node/diagnostic_channel.d.ts

```diff
@@ -0,0 +1,37 @@
+declare module 'node:diagnostic_channel' {
+    export * from 'diagnostic_channel';
+}
+
+/**
+ * @experimental
+ */
+declare module 'diagnostic_channel' {
+    /**
+     * Returns wether a named channel has subscribers or not.
+     */
+    function hasSubscribers(name: string): boolean;
+
+    /**
+     * Gets or create a diagnostic channel by name.
+     */
+    function channel(name: string): Channel;
+
+    type ChannelListener = (name: string, message: unknown) => void;
+
+    /**
+     * Simple diagnostic channel that allows
+     */
+    class Channel {
+        readonly name: string;
```

Comment:
The channel has a [hasSubscribers] property too https://nodejs.org/docs/latest-v15.x/api/diagnostics_channel.html#diagnostics_channel_channel_hassubscribers